home/modules/contribute/pages/extensions-anywhere-footnotes.adoc

```diff
@@ -0,0 +1,110 @@
+= Anywhere Footnotes
+
+One of the AsciiDoc extension extras we run on the site is Anywhere footnotes. 
+This extension allows you to add footnotes to the page, 
+with the block of references appearing anywhere on the page. 
+(Regular AsciiDoc footnotes always appear right at the bottom).
+
+This is very handy for pages where you have multiple tables appearing on the page, like this:
+
+.Sample Table Title
+[cols="1,2,2", options="header"]
+|===
+|ID |Name |Description
+
+|1
+|Product Aafnote:first-block[This is the first footnote]
+|High-quality widget with advanced features{empty}afnote:first-block[This is the second]
```

Comment:
I like the `{empty}` to disambiguate, we should have that throughout to recommend that as best practice.